.github/workflows/matchers/pylint.json

```diff
@@ -0,0 +1,18 @@
+{
+    "problemMatcher": [
```

Comment:
```suggestion
  "problemMatcher": [
```

Can we use 2 spaces for indent?

Reply:
Can we apply the same fix to other lines?

Reply:
Sorry. I will fix the indent.